packages/angular/src/generators/add-linting/lib/buildable-project.ts

```diff
@@ -0,0 +1,12 @@
+import { readProjectConfiguration, type Tree } from '@nx/devkit';
+
+export function isBuildableLibraryProject(
+  tree: Tree,
+  projectName: string
+): boolean {
+  const projectConfig = readProjectConfiguration(tree, projectName);
+
+  return (
+    projectConfig.projectType === 'library' && !!projectConfig.targets?.build
```

Comment:
Would it be better to check for executor name?
Someone may not have named the target "build".

Reply:
This is an internal generator only used by the Angular plugin generators (folks could still use it, but that's like using a private API, not guaranteed to work). We already had this same check for quite a while because our generators always generate the target `build` for buildable libraries.

Reply:
no problem :)